jerry-debugger/jerry-client-ws.py

```diff
@@ -180,11 +174,21 @@ def do_quit(self, args):
         self.quit = True
 
     def do_break(self, args):
-        """ Insert breakpoints on the given lines """
-        self.insert_breakpoint(args)
+        """ Insert breakpoints on the given lines or functions """
+        if args == "":
+            print("Error: Breakpoint index expected")
+        else:
+            set_breakpoint(self.debugger, False, args)
 
     do_b = do_break
 
+    def do_fbreak(self, args):
+        """ Insert breakpoints on the given lines or functions, if not found add the pending list """
+        if args == "":
+            print("Error: Breakpoint index expected")
+        else:
+            set_breakpoint(self.debugger, True, args)
+
```

Comment:
Why fbreak? What does 'f' means? And you can pass the True/False as an argument to insert_breakpoint rather than removing it.